orbit/pkg/luks/luks_linux.go

```diff
@@ -35,13 +37,34 @@ const (
 
 var ErrKeySlotFull = regexp.MustCompile(`Key slot \d+ is full`)
 
+func isInstalled(toolName string) bool {
+	path, err := exec.LookPath(toolName)
+	if err != nil {
+		return false
+	}
+	return path != ""
+}
+
 func (lr *LuksRunner) Run(oc *fleet.OrbitConfig) error {
 	ctx := context.Background()
 
 	if !oc.Notifications.RunDiskEncryptionEscrow {
 		return nil
 	}
 
+	if !isInstalled("cryptsetup") {
+		return errors.New("cryptsetup is not installed")
```

Comment:
Any distros that we know (and want to support) don't have it?

Reply:
I haven't run into this yet, but we're calling `cryptsetup` downstream so figured this would be a good place to exit early.

Reply:
Yep, good idea. Just checking out of curiosity.